resources/quality/ultimaker_s5/um_s5_aa0.8_PLA_Draft_Print.inst.cfg

```diff
@@ -17,17 +17,17 @@ cool_fan_speed_max = =100
 cool_min_speed = 2
 gradual_infill_step_height = =3 * layer_height
 infill_line_width = =round(line_width * 0.65 / 0.75, 2)
-infill_pattern = cubic
+infill_pattern = triangles
 line_width = =machine_nozzle_size * 0.9375
 machine_nozzle_cool_down_speed = 0.75
 machine_nozzle_heat_up_speed = 1.6
 material_final_print_temperature = =max(-273.15, material_print_temperature - 15)
 material_initial_print_temperature = =max(-273.15, material_print_temperature - 10)
 material_print_temperature = =default_material_print_temperature + 10
-prime_tower_enable = False
+material_standby_temperature = 100
```

Comment:
You don't have to add this, because it is in the material profile: https://github.com/Ultimaker/fdm_materials/blob/acc46fbc5435e4deba485ec4eb06829e12de98c9/generic_pla.xml.fdm_material#L90